projects/ion/src/lib/radio/radio.component.html

```diff
@@ -0,0 +1,17 @@
+<form class="ion-input-radio">
+  <input
+    (click)="select()"
+    name="label"
+    id="label"
+    value="label"
```

Comment:
```suggestion
    [name]="label"
    [id]="label"
    [value]="label"
```

check this.

Reply:
Se o id ficar tiver entre [ ] ele para de selecionar quando clica na label